README.md

```diff
@@ -71,6 +71,61 @@ To begin a deployment from scratch, go to the root of your socok8s
 clone and run "./run.sh". This will run each of the seven top-level
 sections of the script in order.
 
+## Reference archiecture and inventories
```

Comment:
Should be 'architecture'. If you make further changes or cleanups to this patch, it'd be great to address typo fixes, but if not we can always tidy it in a follow-up.